### ISSUE TITLE
针对不能在paddle>2.0版本OCR_Dataset数据集样例不能训练问题，报错(InvalidArgument) Input dimension size should be equal to 2, but received dimension size is 1.提出更改

### ISSUE THREAD

**Issue by buganluo14** [GT-ZhangAcer/CaptchaDataset#4]
报错详情：
发生异常: ValueError
(InvalidArgument) Input dimension size should be equal to 2, but received dimension size is 1.
  [Hint: Expected arity(dims) == D, but received arity(dims):2 != D:1.] (at ..\paddle/fluid/framework/eigen.h:35)

提出更改：
在class CTCLoss(pp.nn.Layer)：中的
def forward(self, ipt, label):
            input_lengths = pp.tensor.creation.full([BATCH_SIZE, 1], LABEL_MAX_LEN + 4, "int64")
            label_lengths = pp.tensor.creation.full([BATCH_SIZE, 1], LABEL_MAX_LEN, "int64")
            
            loss = pp.nn.functional.ctc_loss(ipt, label, input_lengths, label_lengths, blank=10)
            return loss
做出修改如下：
def forward(self, ipt, label):
            size = label.shape[0]
            input_lengths = pp.tensor.creation.full([size], LABEL_MAX_LEN + 4, "int64")
            label_lengths = pp.tensor.creation.full([size], LABEL_MAX_LEN, "int64")

            loss = pp.nn.functional.ctc_loss(ipt, label, input_lengths, label_lengths, blank=10)
            return loss
此改动针对CTC的输入大小做了自适应变化。

**Comment by GT-ZhangAcer** [comment]
#5
